core/src/wallet/cosmos/api_impl/CosmosLikeTransactionApi.cpp

```diff
@@ -274,7 +277,7 @@ namespace ledger {
             document.AddMember(kTx, txObject, allocator);
 
             // Set mode
-            // FIXME What mode do we want? (sync|async|block)
+            // TODO What mode do we want? (sync|async|block)
             vString.SetString("async", static_cast<SizeType>(5), allocator);
             document.AddMember(kMode, vString, allocator);
```

Comment:
This is related to the GaiaExplorer, we'll have to move that later

Reply:
> Let's test this code and see if the signature is a problem

I confirm that it doesn't work for signed transactions, see test `CosmosTransactionTest.ParseRawSignedMsgSendTransaction` that fails.

Reply:
I got it from here:
https://github.com/cosmos/cosmos-sdk/blob/2e42f9cb745aaa4c1a52ee730a969a5eaa938360/x/auth/client/rest/broadcast.go#L13-L16
But you think not all explorers will use the same syntax?...

Reply:
They don't have to. But most likely the only explorer will be the gaia one anyway. We'll move that if we ever need to support another explorer